packages/oruga/src/components/collapse/tests/SimpleCollapse.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+import { ref } from "vue";
+
+import OCollapse from "../Collapse.vue";
+
+const isOpen = ref(0);
+
+const collapses = [
+    {
+        title: "Title 1",
+        text: "Text 1",
+    },
+    {
+        title: "Title 2",
+        text: "Text 2",
+    },
+    {
+        title: "Title 3",
+        text: "Text 3",
+    },
+];
+</script>
+
+<template>
```

Comment:
Maybe you can convert this component into an inline component, like you can see in carousel.axe.test.ts. so we reduce the need for an extra file and have everthing fo the test in one file :)